core/src/main/java/org/infinispan/topology/CacheStatusResponse.java

```diff
@@ -35,6 +38,18 @@ public CacheStatusResponse(CacheJoinInfo cacheJoinInfo, CacheTopology cacheTopol
       this.current = current;
    }
 
+   public static CacheStatusResponse empty() {
+      return EMPTY;
+   }
+
+   public boolean isEmpty() {
+      return cacheJoinInfo == null
```

Comment:
Nitpick. Is the `EMPTY` instances always used when this is true? If so, we can simplify to `return this == EMPTY`

Reply:
Yeah, the `EMPTY` is supposed to be all null. But since I didn't want to mess with changing the externalizer, it creates a new instance, so using the `==` doesn't work, and neither does `.equals` as we don't override the method :frowning_face: